packages/itwin/property-grid/src/components/FilteringPropertyGrid.tsx

```diff
@@ -177,6 +177,9 @@ export const FilteringPropertyGrid = (
     <>
       <VirtualizedPropertyGridWithDataProvider
         {...props}
+        minLabelWidth={10}
+        minValueWidth={10}
+        actionButtonWidth={props.actionButtonWidth ?? props.actionButtonRenderers ? 90 : 0}
```

Comment:
Maybe instead of 90 we can pass `undefined` so that `VirtualizedPropertyGrid` default would be used?

Reply:
fixed